### ISSUE TITLE
64 bit applications widget

### ISSUE THREAD

**Issue by gmarnin** [munkireport/munkireport-php#1020]
It would be helpful to have a widget showing which apps are 64 bit and which are not.

**Comment by poundbangbash** [comment]
Testing and reporting on the PR at https://github.com/munkireport/munkireport-php/pull/1021 would be beneficial to this issue.

To checkout the PR branch:
```
git remote add tuxudo https://github.com/tuxudo/munkireport-php
git fetch tuxudo
git checkout 32-bit-widget
```

**Comment by bochoven** [comment]
I merged the PR, but I hope you did some testing..

**Comment by bochoven** [comment]
Should be addressed in 3.1.0
